mappings/net/minecraft/item/ItemGroups.mapping

```diff
@@ -39,6 +40,7 @@ CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
 	METHOD m_jjegmdcl generateMaxEnchantedBookEntries (Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;Lnet/minecraft/unmapped/C_vtbxyypo;Ljava/util/Set;Lnet/minecraft/unmapped/C_lnokcayd$C_tndlmady;Lnet/minecraft/unmapped/C_czxxrbcp;)V
 		ARG 1 enchantmentLookup
 		ARG 2 targets
+		ARG 4 featureFlagBits
```

Comment:
```suggestion
		ARG 4 featureFlags
```

Reply:
Fixed this for all 3 mappings that I set this for!